### DISCUSSION TITLE
Render Targets

### DISCUSSION THREAD

**Discussion by niziul** [scripthookvdotnet/scripthookvdotnet#1321] [Ideas]
Would it be possible to add an implementation to handle render targets in SHVDN? I think it's something that could be useful for the community.

**Comment by kagikn** [comment]
Demanding enough for me to implement, but I think I should research how the game stores render target stuff so I can come up with better design. Since I think sophisticated chrono stuff may be more demanding and will be more difficult to design, render target classes will be implemented after #1281.

**Reply by niziul** [reply]
I agree, it would be very nice.
